debian/control

```diff
@@ -653,12 +661,13 @@ Description: RocksDB storage engine for MariaDB
 
 Package: mariadb-plugin-columnstore
 Architecture: amd64 i386
-Depends: mariadb-server-10.5 (= ${binary:Version}),
-         net-tools,
-         binutils,
-         libsnappy1 | libsnappy1v5,
+Depends: binutils,
+         expect,
```

Comment:
All active scripts don't use expect. It is used by obsolete scripts we are removing soon. Hopefully next release.

Reply:
We can remove them then?